### DISCUSSION TITLE
Installing the latest version of Klipperscreen shows insufficient privileges

### DISCUSSION THREAD

**Discussion by yang6991906** [KlipperScreen/KlipperScreen#1414] [Q&A] [answered]
After installing the latest version of Klipperscreen (v0.4.2-37), there is an issue where a prompt "insufficient privileges" appears when opening the network panel. May I know how to solve this problem. My system is Ubuntu 24.04 Arm64 architecture. The following is the network version number:
##dpkg -l | grep network-manager
ii  network-manager   1.46.0-1ubuntu2       arm64        network management framework (daemon and userspace tools)
ii  network-manager-pptp    1.2.12-3build2       arm64        network management framework (PPTP plugin core)

**Comment by yang6991906** [comment]
After opening the network, a prompt will appear      "Unexpected error: org.freedesktop.NetworkManager. wifi.scan request failed: not authorized"

**Comment by thijstriemstra** [comment, marked as answer]
(re-)install klipperscreen, this should be fixed now.
